libs/transloco/src/lib/scope-resolver.ts

```diff
@@ -8,7 +15,14 @@ type ScopeResolverParams = {
 };
 
 export class ScopeResolver {
-  constructor(private service: TranslocoService) {}
+  readonly config: TranslocoConfig;
+
+  constructor(
+    private service: TranslocoService,
+    @Inject(TRANSLOCO_CONFIG) userConfig: PartialTranslocoConfig = {},
```

Comment:
You can't use angular injection as this is a pure JS class and isn't an injectable.

Reply:
Fixed